add docs for netboot (#18)
rancher-sandbox/cos-toolkit-docs: content/en/docs/Getting started/booting.md

cos-deploy --docker-image $IMAGE
```

## Booting from network

> You can boot a cOs squash fs by using a native iPXE implementation on your

Reviewer: ```suggestion
You can boot a cOS squashfs by using a native iPXE implementation on your
```

system or by inserting a custom build iPXE iso by using your either your servers
management web-interface (ILO, iDrac, ...) or using a
[redfish](https://www.dmtf.org/standards/redfish) library or its vendor specific implementation (e.g. [ilorest](https://hewlettpackard.github.io/python-redfish-utility/)). 

To boot from IPXE you need to extract the squashfs from

> [https://quay.io/repository/costoolkit/releases-green?tab=tags](https://quay.io/repository/costoolkit/releases-green?tab=tags).

Reviewer: ```suggestion
[images](https://quay.io/repository/costoolkit/releases-green?tab=tags) with the `cos-img-recovery-` prefix.
```

By pulling the image, saving it using docker save, then unpack the single layer
in the file to an extra directory and then run 

```bash
$> mksquashfs <path_to_folder> <output_path>/root.squashfs
```

**Note:**

> The squashfs file can also be extracted using the following `luet` command.

Reviewer: ```suggestion
The squashfs file can also be extracted using the following `docker` command:
```
